js-miniapp-sdk/README.md

```diff
@@ -110,6 +110,4 @@ function getId() {
 
 ## Changelog
 
-### v1.0.0 (2020-5-13)
-
-- Initial release.
+See the full [CHANGELOG](CHANGELOG.md).
```

Comment:
I think this shouldn't be separated into a different file. The README is published to Github pages at https://rakutentech.github.io/js-miniapp/docs/1.0/ (and it also gets published to [npmjs](https://www.npmjs.com/package/js-miniapp-sdk), so the Changelog needs to be available at the same location as the user guide. 

The link here to `CHANGELOG.md` could be updated to point to the full github.com url, but I think it's not as nice as having the change log built-in to the user guide. Having it in the User Guide allows the reader to more easily search (via cmd+f) and to more easily navigate between the changelog/guide. Also, it's sometimes necessary to put links in the changelog to specific parts of the user guide, and this is easier to do if they are in the same page.

My alternative suggestion is this:
- Keep the Changelog at the bottom of the User Guide
- Add a Table of Contents at the top of the User Guide to make navigation to the Changelog easier
- Once the Changelog gets too long, we can collapse the entries for older versions so that they're expandable by clicking (via JavaScript), or we can separate old versions into a different file.
- Update the [github releases](https://github.com/rakutentech/js-miniapp/releases) with changes for each release.

Reply:
Indeed I like your suggestion @corycaywood

Reply:
As we spoke, updated the link to the absolute value of the URL.